spec/controllers/hq/orphans_controller_spec.rb

```diff
@@ -14,7 +14,7 @@
 
   specify '#index' do
     expect(Orphan).to receive(:paginate).with(page: "2")
-      .and_return(orphans.paginate(per_page: 2, page: 1))
+                          .and_return(orphans.paginate(per_page: 2, page: 1))
```

Comment:
don't make unnecessary changes like these, it might cause merge conflicts for others.